<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>746</width>
    <height>550</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="tab">
       <attribute name="title">
        <string>Tab 1</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout">
        <item>
         <widget class="QTextEdit" name="edQFile"/>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="tab_2">
       <attribute name="title">
        <string>Tab 2</string>
       </attribute>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <widget class="QTextEdit" name="edQTextStream"/>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="toolButtonStyle">
    <enum>Qt::ToolButtonTextUnderIcon</enum>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actQFileOpen"/>
   <addaction name="actQFileSave"/>
   <addaction name="separator"/>
   <addaction name="actQTextOpen"/>
   <addaction name="actQTextSave"/>
   <addaction name="separator"/>
   <addaction name="actExit"/>
  </widget>
  <action name="actQFileOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/7_2.png</normaloff>:/images/images/7_2.png</iconset>
   </property>
   <property name="text">
    <string>QFile直接打开</string>
   </property>
   <property name="toolTip">
    <string>QFile直接打开</string>
   </property>
  </action>
  <action name="actQFileSave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/7_1.png</normaloff>:/images/images/7_1.png</iconset>
   </property>
   <property name="text">
    <string>QFile另存为</string>
   </property>
  </action>
  <action name="actQTextOpen">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/7_3.png</normaloff>:/images/images/7_3.png</iconset>
   </property>
   <property name="text">
    <string>QTextSteam打开</string>
   </property>
  </action>
  <action name="actQTextSave">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/7_4.png</normaloff>:/images/images/7_4.png</iconset>
   </property>
   <property name="text">
    <string>QTextSteam另存为</string>
   </property>
  </action>
  <action name="actExit">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/images/images/7_quit.png</normaloff>:/images/images/7_quit.png</iconset>
   </property>
   <property name="text">
    <string>退出</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>actExit</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>372</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
